src/vision/src/RosVision.cpp

```diff
@@ -91,15 +116,25 @@ void RosVision::imageCallback(const sensor_msgs::ImageConstPtr &msg) {
 }
 
 void RosVision::createWindow() {
-    namedWindow(inputWindow, CV_WINDOW_AUTOSIZE);
-    namedWindow(ipmOutputWindow, CV_WINDOW_AUTOSIZE);
-    namedWindow(filterOutputWindow, CV_WINDOW_AUTOSIZE);
-}
-
-void RosVision::deleteWindow() {
-    destroyWindow(inputWindow);
-    destroyWindow(ipmOutputWindow);
-    destroyWindow(filterOutputWindow);
+    // Create one big mat for all our images
+    cv::Size main_window_size(displayWindowWidth, displayWindowHeight);

```

Comment:
displayWindowWidth and displayWindowHeight doesn't determine the actual overview window size, just the size of the Mat which holds the image collage. Since we create the window with the CV_WINDOW_NORMAL flag, which allows for size changes (compared to CV_WINDOW_AUTOSIZE which makes the window the size of the underlying Mat with no window resize). I think we can deprecate these two parameters and have them autoset by the size of the image.